test/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:jessie
```

Comment:
I'm not fluent in `Dockerfile`, but this makes sense and seems to cover the bases!